lib/usdUfe/ufe/trf/XformOpUtils.h

```diff
@@ -27,48 +28,62 @@
 
 #include <vector>
 
-namespace MAYAUSD_NS_DEF {
-namespace ufe {
+namespace USDUFE_NS_DEF {
 
+USDUFE_PUBLIC
```

Comment:
These are used by the Maya specific transform3d code left behind in MayaUsd. So needed to export the functions.